cypress/utils/shippingUtils.js

```diff
@@ -0,0 +1,71 @@
+import ShippingMethod from "../apiRequests/ShippingMethod";
+import Warehouse from "../apiRequests/Warehouse";
+class ShippingUtils {
+  shippingMethodId;
+  shippingZoneId;
+  warehouseId;
+
+  createShipping(channelId, name, address, price) {
+    const shippingMethod = new ShippingMethod();
+    const warehouse = new Warehouse();
+    return shippingMethod
```

Comment:
We generally try to avoid then chaining to avoid callback hell - feel free to check [this](https://blog.hellojs.org/asynchronous-javascript-from-callback-hell-to-async-and-await-9b9ceb63c8e8) out